docs/best-practices/continuous-integration/github.md

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions Integration
+
+Use [GitHub Actions runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) running on a macOS machine
+to build using fastlane.
+
+## Repository setup
+
+First create a `Gemfile` in the root of your project with the following content:
+
+```ruby
+source 'https://rubygems.org'
+
+gem 'fastlane'
+```
+
+Add a `.github/workflows/build-ios-app.yml` file to trigger _fastlane_.
+
+```yml
+name: build-ios-app
+on:
+  push:
+    branches:
+      - 'master'
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: fastlane beta
+        env:
+          MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+```
+
+See [Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)
+for more information on how this file works.
+
+## Setting up the lanes
+
+```ruby
```

Comment:
Again, I think it's worth mentioning this is just a sample fastfile config

```suggestion
Here's an example of a `Fastfile` with a lane that runs _match_, builds the app, and uploads to TestFlight:

```ruby
```